### PR TITLE
Update SRG-OS-000064-GPOS-00033 for RHEL9 STIG 

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

* Update SRG-OS-000064-GPOS-00033 for RHEL9 STIG 
* Fix a spelling error in some titles

#### Rationale:

RHEL9 STIG

### CODE REVIEW

File: controls/srg_gpos/SRG-OS-000064-GPOS-00033.yml [hunk 1]
```diff
@@ -22,16 +22,7 @@ controls:
...
-            - audit_rules_unsuccessful_file_modification_open_by_handle_at_o_trunc_write
-            - audit_rules_unsuccessful_file_modification_open_by_handle_at_rule_order
-            - audit_rules_unsuccessful_file_modification_open_o_creat
-            - audit_rules_unsuccessful_file_modification_open_o_trunc_write
-            - audit_rules_unsuccessful_file_modification_open_rule_order
             - audit_rules_unsuccessful_file_modification_openat
-            - audit_rules_unsuccessful_file_modification_openat_o_creat
-            - audit_rules_unsuccessful_file_modification_openat_o_trunc_write
-            - audit_rules_unsuccessful_file_modification_openat_rule_order
             - audit_rules_unsuccessful_file_modification_rename
```
Comment (Mab879) [inline controls/srg_gpos/SRG-OS-000064-GPOS-00033.yml:26]: In the newest STIGs these are combined, how do we want to handle this?

Comment (ggbecker) [reply controls/srg_gpos/SRG-OS-000064-GPOS-00033.yml:26]: We will push them to keep rules separate, if they ultimately reject we can reevaluate.

**Comment by ggbecker** [comment]
@Mab879 let's drop all the `audit_rules_unsuccessful_*` rules from the SRGs.

File: shared/macros/ocil.jinja [hunk 2]
```diff
@@ -129,6 +143,21 @@ ocil: |
...
 {{%- endmacro %}}
 
+{{#
+    OCIL and OCIL clause for adding a unsuccessful syscall to audit logs
+
+:param syscall: The syscall to audit
+:type syscall: str
+
+#}}
+{{% macro complete_ocil_entry_audit_unsuccessful_syscall(syscall) -%}}
```
Comment (ggbecker) [inline shared/macros/ocil.jinja:153]: Where do you plan to use this macro? I don't see it being used in this pull request.

Comment (Mab879) [reply shared/macros/ocil.jinja]: Good catch, I move some rule to use this now.

**Pull request merged**